File: app/src/main/java/dev/ragnarok/fenrir/db/model/entity/CopiesEntity.java
package dev.ragnarok.fenrir.db.model.entity;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import dev.ragnarok.fenrir.db.model.IdPairEntity;


public class CopiesEntity {

    @SerializedName("count")
    private int count;
    @SerializedName("pairDbos")
    private List<IdPairEntity> pairDbos;

    public int getCount() {
        return count;
    }

    public CopiesEntity setCount(int count) {
        this.count = count;
        return this;
    }

    public List<IdPairEntity> getPairDbos() {
        return pairDbos;
    }

    public CopiesEntity setPairDbos(List<IdPairEntity> pairDbos) {
        this.pairDbos = pairDbos;
        return this;
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/db/model/entity/CopiesEntity.kt
package dev.ragnarok.fenrir.db.model.entity

import androidx.annotation.Keep
import dev.ragnarok.fenrir.db.model.IdPairEntity
import kotlinx.serialization.Serializable

@Keep
@Serializable
class CopiesEntity {
    var count = 0
        private set

    var pairDbos: ArrayList<IdPairEntity>? = null
        private set

    fun setCount(count: Int): CopiesEntity {
        this.count = count
        return this
    }

    fun setPairDbos(pairDbos: ArrayList<IdPairEntity>?): CopiesEntity {
        this.pairDbos = pairDbos
        return this
    }
}